IPython/core/completer.py

```diff
@@ -9,7 +9,7 @@
 Syntax like magics.
 
 Latex and Unicode completion
-============================
+====================:========
```

Comment:
Probably not intentional :)

Otherwise, LGTM.

Reply:
Doh.. force-pushed.